gremlin-console/src/main/java/org/apache/tinkerpop/gremlin/console/jsr223/ConnectionHelper.java

```diff
@@ -16,28 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.tinkerpop.gremlin.console.commands
+package org.apache.tinkerpop.gremlin.console.jsr223;
 
-import org.apache.groovy.groovysh.CommandSupport
-import org.apache.groovy.groovysh.Groovysh
-import org.apache.tinkerpop.gremlin.console.Mediator
+import static org.apache.tinkerpop.gremlin.process.traversal.AnonymousTraversalSource.traversal;
 
-/**
- * Submit a script to a Gremlin Server instance.
- * @author Stephen Mallette (http://stephen.genoprime.com)
- */
-class SubmitCommand extends CommandSupport {
-
-    private final Mediator mediator
-
-    public SubmitCommand(final Groovysh shell, final Mediator mediator) {
-        super(shell, ":submit", ":>")
-        this.mediator = mediator
+public class ConnectionHelper {
```

Comment:
Nit: I don't think this class belongs in jsr223 as it doesn't do anything to the ScriptEngine. Maybe move it?